test/errors/fullImportPath-detailed.good

```diff
@@ -1,5 +1,5 @@
 ─── error in fullImportPath.chpl:9 [UseImportUnknownMod] ───
-  Cannot find module 'ToLookFor' for 'import' statement.
+  Cannot find module named 'ToLookFor'.
   In the following 'import' statement:
```

Comment:
Again, this is a pre-existing condition, but "In the following 'import' statement isn't a complete sentence and feels like an obvious thing to combine into the previous sentence if that were easy to do.

Reply:
The reason it's not printed that way is because the words "In the following import statement:" are absent in brief mode (along with the code snippet that prints the import statement). It seemed to make sense to separate the not-always-there text from the always-there text by a newline.

Reply:
I understand (now) that that's the explanation for why it's this way, but... it seems like it makes the error read really choppily and poorly to me.  Even if we're constrained  to keep this formatting, could we at least use a complete sentence to avoid giving the impression that the `.\nI` were added arbitrarily?

(And... oh wait, I wasn't blind and there was a `Cannot` in a new .good file... but now you've reminded me why that is).